### ISSUE TITLE
Game data update required

### ISSUE THREAD

**Issue by Gnomesenpai** [akowald/StopThatTank#26]
The recent TF2 update has caused a server crash to occur when players become Giant demoman robots

https://crash.limetech.org/yqv4pdbqmckx

https://crash.limetech.org/4eb45dxj2ley

**Comment by chocochocotorta** [comment]
Try with this one.
I'm not sure if i updated everything, but i think it works pretty normal.

[tank.txt](https://github.com/akowald/StopThatTank/files/8980716/tank.txt)

**Comment by Gnomesenpai** [comment]
Thanks, I will give it a try and report back!

**Comment by NickCloudAT** [comment]
@Procustes 
How does updating the gamedata work?
After the recent updates it seems to be broken again..

**Comment by Gnomesenpai** [comment]
Yeah it worked for a bit, but the most recent update has really messed something up #27
